nodewatcher/nodewatcher.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.6
+#!/usr/bin/env python
```

Comment:
why are we changing this only in nodewatcher?

Reply:
changed in both